.eslintrc.js

```diff
@@ -12,6 +12,7 @@ module.exports = {
 
   rules: {
     semi: ['error', 'always'],
+    'no-console': 'off',
```

Comment:
I'm going to add a comment here to remove this. What you can do for eslint not to bother you is in the `vue.config.js`, add a line as `lintOnSave: false` and you are good to go